benches/mod.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(test)]
+
+extern crate test;
+
+use p14::add;
+
+use test::{black_box, Bencher};
+
+#[bench]
+
```

Comment:
You should add benches for the Finbonacci functions. Also the empty line between `#[bench]` and `fn` is not needed.